engines/scumm/saveload.cpp

```diff
@@ -1137,6 +1137,11 @@ void ScummEngine::saveOrLoad(Serializer *s) {
 	if (s->isLoading() && s->getVersion() < VER(14))
 		upgradeGfxUsageBits();
 
+	// When loading, reset the setShakePos. Fixes bug #7141 Partially
```

Comment:
@bluegr Bug #7141 can be split up into 2 issues. The shakePos is not reset on loading a saved game, and when shakePos is not 0, mousepointer is not handled correctly. The second part is also present during normal gameplay, but less visible.

@wjp I changed the commit messages/comments a bit.